tests/test_views.py

```diff
@@ -77,6 +97,27 @@ def setUp(self):
         self.user.save()
         self.client.login_user(self.user)
 
+        self.mocked_datetime = patch(
```

Comment:
I think there's two better ways to do this.

1. The mock library conveniently has an example of how to make a function of the `datetime` module without a dummy class [here](https://docs.python.org/3/library/unittest.mock-examples.html#partial-mocking)!
2. You could also consider using the `freezegun` library, which we use in several places. [Here's](https://github.com/edx/learning-assistant/blob/93d620e49447c54d6e70f7816c6823dcc5eb2534/tests/test_api.py#L527-L533) an example in this repository.